alma/alma-installments-prestashop
# feat: Remove insurance in cart when deactivated

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-2003/ps-remove-insurance-in-cart-when-deactivated)

### Code changes


Change state insurance product according insurance config state
Delete insurance product in cart if insurance is disable

### How to test

_As a reviewer, you are encouraged to test the PR locally._


Add insurance product in cart
Disable insurance in the configuration
See the insurance product is disable in the product list in back office
See the insurance product is deleted in the same cart

### Checklist for authors and reviewers




### Non applicable
